sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/DisposableDataFeed.cs

```diff
@@ -25,39 +25,35 @@ public class DisposableDataFeed : IAsyncDisposable
         /// Initializes a new instance of the <see cref="DisposableDataFeed"/> class.
         /// </summary>
         /// <param name="adminClient">The client to use for deleting the data feed upon disposal.</param>
-        /// <param name="id">The identifier of the data feed this instance is associated with.</param>
-        private DisposableDataFeed(MetricsAdvisorAdministrationClient adminClient, string id)
+        /// <param name="dataFeed">The data feed this instance is associated with.</param>
+        private DisposableDataFeed(MetricsAdvisorAdministrationClient adminClient, DataFeed dataFeed)
         {
             _adminClient = adminClient;
-            Id = id;
+            DataFeed = dataFeed;
         }
 
         /// <summary>
-        /// The identifier of the data feed this instance is associated with.
+        /// The data feed this instance is associated with.
         /// </summary>
-        public string Id { get; }
+        public DataFeed DataFeed { get; }
 
         /// <summary>
         /// Creates a data feed using the specified <see cref="MetricsAdvisorAdministrationClient"/>.
-        /// A <see cref="DisposableDataFeed"/> instance is returned, from which the ID of the created
-        /// data feed can be obtained. Upon disposal, the associated data feed will be deleted.
+        /// A <see cref="DisposableDataFeed"/> instance is returned, from which the created data feed
+        /// can be obtained. Upon disposal, the associated data feed will be deleted.
         /// </summary>
         /// <param name="adminClient">The client to use for creating and for deleting the data feed.</param>
-        /// <param name="hook">Specifies how the created <see cref="DataFeed"/> should be configured.</param>
-        /// <returns>A <see cref="DisposableDataFeed"/> instance from which the ID of the created data feed can be obtained.</returns>
+        /// <param name="dataFeed">Specifies how the created <see cref="Models.DataFeed"/> should be configured.</param>
+        /// <returns>A <see cref="DisposableDataFeed"/> instance from which the created data feed can be obtained.</returns>
         public static async Task<DisposableDataFeed> CreateDataFeedAsync(MetricsAdvisorAdministrationClient adminClient, DataFeed dataFeed)
         {
             DataFeed createdDataFeed = await adminClient.CreateDataFeedAsync(dataFeed);
-
-            Assert.That(createdDataFeed, Is.Not.Null);
-            Assert.That(createdDataFeed.Id, Is.Not.Null.And.Not.Empty);
```

Comment:
Wanted to remove validation from the disposable types. Move it to the actual `DataFeedLiveTests` file.